js/store/environment/actions.js

```diff
@@ -0,0 +1,26 @@
+import moment from 'moment';
```

Comment:
I think we can remove the entire `environment` store, not using date or location if I'm reading correctly

Reply:
And also perhaps remove moment.js dependency. It's big